src/org/usfirst/frc/team467/robot/RobotMap.java

```diff
@@ -131,45 +131,39 @@ public static void init(RobotID id) {
 
 			HAS_GRABBER = true;
 			GRABBER_INVERT = false;
-			HAS_CAMERA = false;
-
+			HAS_CAMERA = false;           
 			GRABBER_L_CHANNEL = 0;
 			GRABBER_R_CHANNEL = 1;
-			OPTICAL_CHANNEL = 5;
-
+			OPTICAL_CHANNEL = 5;            
 			// TODO Assign values to the game piece variables, and make more as appropriate
 			HAS_ELEVATOR = true;
 			ELEVATOR_MOTOR_CHANNEL = 7;
-			ELEVATOR_BOTTOM_TICKS = 764;
-			ELEVATOR_FLOOR_HEIGHT = 747;
-			ELEVATOR_SWITCH_HEIGHT = 636;
-			ELEVATOR_LOW_SCALE_HEIGHT = 468;
-			ELEVATOR_TOP_TICKS = 357;
-
+			ELEVATOR_BOTTOM_TICKS = 839;
+			ELEVATOR_FLOOR_HEIGHT = 822;
+			ELEVATOR_SWITCH_HEIGHT = 711;
+			ELEVATOR_LOW_SCALE_HEIGHT = 543;
+			ELEVATOR_TOP_TICKS = 443;           
```

Comment:
I think these elevator ticks numbers are good to keep. But I also see a lot of unwanted whitespace changes, too. Please back out the bogus whitespace changes.